website/archiver/decorators.py

```diff
@@ -6,6 +6,7 @@
 from website.archiver import ARCHIVER_NETWORK_ERROR
 from website.archiver import signals
 
+
```

Comment:
This is fine, but we explicitly ignore the "2-blank-line rule" in our setup.cfg: https://github.com/CenterForOpenScience/osf.io/blob/eb9a82717b2a4ad8acb5b1650ab79e73c7f324b7/setup.cfg#L6 .